skyportal/tests/api/test_bokeh_plots.py

```diff
@@ -0,0 +1,177 @@
+import os
+from skyportal.tests import api
+import numpy as np
+import datetime
+from bokeh.util import serialization
+
+
+def black_body(la, temp):
+    """
+    get the amount of radiation expected from a black body
+    of the given temperature "temp", at the given wavelengths "la".
+    :param la: float array or scalar
```

Comment:
We don't use this style of docstring in skyportal; please use numpydoc format

Reply:
Sorry. My IDE got the best of me. I'll fix it.